website/docs/r/log_analytics.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_analytics"
+sidebar_current: "docs-azurerm-resource-log-analytics"
```

Comment:
(as above) - could we include `-oms-` in this - to be able to highlight both the title and the resource name in the sidebar?

Reply:
Done